### DISCUSSION TITLE
Where should we deploy the Shiny app?

### DISCUSSION THREAD

**Discussion by thisisnic** [thisisnic/shinyarrowpi#3] [Ideas]
Where should we deploy the Shiny app?

**Comment by sellorm** [comment]
We could go the whole "production" way and deploy it to RStudio Connect.

**Reply by thisisnic** [reply]
Sounds good to me; I think doing something like that would allow us to squeeze the most amount of juice out of this toy project in terms of it serving as a potential example of various things.

**Reply by sellorm** [reply]
Cool. I have a license we can use.

If the app is not too hungry, I'll set up an ec2 instance for us to use nearer the time.
